app/src/main/kotlin/com/twain/interprep/data/model/Interview.kt

```diff
@@ -2,11 +2,47 @@ package com.twain.interprep.data.model
 
 import androidx.room.Entity
 import androidx.room.PrimaryKey
+import androidx.room.TypeConverter
 import java.util.Calendar
 
 @Entity
 data class Interview (
     @PrimaryKey(autoGenerate = true)
     var interviewId: Int = 0,
     var interviewDateTime: Long = Calendar.getInstance().timeInMillis,
-)
\ No newline at end of file
+)
+
```

Comment:
I was trying to put Interview here, but I didn't completely figure how to store `Calendar` object to the database. To focus on the components, I temporarily move the `Interview` data class to `InterviewCard.kt`

Reply:
Check the class DateConverter in the code. I have updated the implementation and right way of storing Date object in Room DB.